rules/falco_rules.yaml

```diff
@@ -2646,13 +2646,15 @@
 - list: k8s_client_binaries
   items: [docker, kubectl, crictl]
 
+# You can overwrite this macro to avoid false positives.
```

Comment:
Please remove the first line

Reply:
Thank you for your review.
I removed it.